lib/services/auth.dart

```diff
@@ -4,41 +4,41 @@ import 'package:firebase_auth/firebase_auth.dart';
 class AuthMethods {
   final FirebaseAuth _auth = FirebaseAuth.instance;
 
-  User _userFromFirebaeUser(FirebaseUser user) {
+  User _userFromFirebaseUser(FirebaseUser user) {
     return user != null ? User(userId: user.uid) : null;
   }
 
-  Future signInWithEmailAndPassword(String email, String password) async {
+  Future SignInWithEmailAndPassword(String email, String password) async {
```

Comment:
lowerCamelCase function naming, and you should consider to use optional parameters with @required annotation. It will improve your code readability